include/nccl_ofi_tracepoint.h

```diff
@@ -9,6 +9,24 @@
 #include "config.h"
 #include "tracing_impl/lttng.h"
 
+/***** SENDRECV PROTOCOL *****/
```

Comment:
why does it make sense to have different protocols?

Reply:
I think it makes sense because
1. With NVTX, we need to keep range identifiers in protocol-specific structures (e.g., `rdma_send_comm_t`), which means at some level we need a different macro to extract the identifier from the correct structure tree.
2. Most of the events in `nccl_ofi_tracepoint.h` aren't relevant to SENDRECV protocol
3. I didn't include support for NVTX for SENDRECV protocol in this PR, and I was hoping to escape that responsibility for some time.

Reply:
I still don't see how to make these generic, for the reasons above.

Reply:
I won't block on this.